src/navigation/reroute.ts

```diff
@@ -37,21 +45,21 @@ export function triggerAppChange() {
 }
 
 export function reroute(
-  pendingPromises = [],
-  eventArguments,
-  silentNavigation = false
-) {
+  pendingPromises: WaitingPromises[] = [],
+  eventArguments: EventArguments = undefined,
```

Comment:
The new default value for this argument is a runtime change necessary for typescript to be ok on line 36